Documentation/project-docs/standard-platform.md

```diff
@@ -41,7 +41,7 @@ In general, class libraries which target a lower .NET Platform Standard version
 
 | Target Platform Name | Alias |  |  |  |  |  |
 | :---------- | :--------- |:--------- |:--------- |:--------- |:--------- |:--------- |
-|.NET Platform Standard | dotnet | 5.1 | 5.2 | 5.3 | 5.4 | 5.5 |
+|.NET Platform Standard | netstandard | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 |
```

Comment:
yea std didn't go over well. A few more characters wont hurt and it's clear what it represents.